moskito-core/src/main/java/net/anotheria/moskito/core/config/tagging/CustomTag.java

```diff
@@ -0,0 +1,66 @@
+package net.anotheria.moskito.core.config.tagging;
+
+import org.configureme.annotations.Configure;
+import org.configureme.annotations.ConfigureMe;
+
+/**
+ * User defined tag.
+ * @author strel
+ */
+@ConfigureMe
+public class CustomTag {
+
+    /**
+     * Tag name.
+     */
+    @Configure
+    private String name;
+
+    /**
+     * Tag value 'container' name (for example session or request attribute).
+     * All possible prefixes are listed in {@link TagPrefix}.
+     */
+    @Configure
+    private String prefix;
```

Comment:
why do we have prefix as separate value? I think prefix means something in the same attribute, like session.attributeName not {"prefix":"session", "name":"attributeName"}